readthedocs/proxito/views/mixins.py

```diff
@@ -270,7 +270,7 @@ def _serve_file_from_python(self, request, path, storage):
         return serve(request, path, root_path)
 
     def _serve_401(self, request, project):
-        res = render(request, "401.html")
+        res = render(request, "errors/401.html")
```

Comment:
If this is a proxito specific error, we should keep it in `errros/proxito/401.html` to disambiguate the error templates.

Reply:
I just saw https://github.com/readthedocs/ext-theme/pull/388

This seems fine for now too, and we can clean up these paths later. If possible, it would be best to separate proxito errors from other potential error templates.

Reply:
Agreed with Anthony. Distinguish proxito from application templates if possible would be great 👍🏼